src/openrct2/peep/Peep.cpp

```diff
@@ -82,186 +82,195 @@ static void peep_128_tick_update(Peep* peep, int32_t index);
 static void peep_release_balloon(Guest* peep, int16_t spawn_height);
 // clang-format off
 
+// Flags used by PeepThoughtToActionMap
+enum PeepThoughtToAction : uint8_t
+{
+    PEEP_THOUGHT_ACTION_NO_FLAGS = 0,
+    PEEP_THOUGHT_ACTION_FLAG_1 = (1 << 0),
+    PEEP_THOUGHT_ACTION_FLAG_2 = (1 << 1),
+    PEEP_THOUGHT_ACTION_FLAG_3 = (1 << 2),
+};
+
 /** rct2: 0x00981DB0 */
 static struct
 {
     PeepActionType action;
-    uint8_t flags;
+    PeepThoughtToAction flags;
```

Comment:
I'd call this `PeepThoughtToActionFlag`. After all, this field does not contain the entire peep-thought-to-action map.